Swiftfin tvOS/Views/VideoPlayer/VideoPlayer.swift

```diff
@@ -83,6 +96,10 @@ struct VideoPlayer: View {
                     .scrubbedSeconds = Int(CGFloat(videoPlayerManager.currentViewModel.item.runTimeSeconds) * newValue)
             }
         }
+        .onDisappear(perform: {
+            videoPlayerManager.proxy.stop()
+            router.dismissCoordinator()
```

Comment:
This causes a crash because you are attempting to dismiss a coordinator when it has already been dismissed.